src/main/java/org/opentripplanner/middleware/controllers/api/LogController.java

```diff
@@ -24,7 +24,7 @@
 import static com.beerboy.ss.descriptor.EndpointDescriptor.endpointPath;
 import static com.beerboy.ss.descriptor.MethodDescriptor.path;
 import static org.opentripplanner.middleware.auth.Auth0Connection.isUserAdmin;
-import static org.opentripplanner.middleware.utils.DateTimeUtils.YYYY_MM_DD;
+import static org.opentripplanner.middleware.utils.DateTimeUtils.DEFAULT_DATE_FORMAT_PATTERN;
```

Comment:
In my view, this variable just became harder to read with this name change.

Reply:
I disagree and believe it has resulted in a more explicit meaning with intended usage conveyed through its name.